### ISSUE TITLE
An invalid character in form field forces bad formatting

### ISSUE THREAD

**Issue by TLMurray23** [holtwick/pdfify#90]
Please describe how the error can be reproduced:
1) Have a PDF with a form field.
2) From somewhere else, put something on the clipboard of an oddball font.
3) Paste it into a field.

Problem: You're stuck with the formatting.
Problem: Most apps adhere to Apple's use of Shift+Option+Command+V to "paste as plain text into the current format" -- but PDFify does not do that.

There is a video at https://techknowledgecorp.wistia.com/medias/jrbnuj08g9 

App Version Info:
de.holtwick.mac.PDFify@3.8.1+275 YPN83VWYDN2ZX1WXQ59EG7TR48

**Comment by holtwick** [comment]
Thanks for the video. 

In PDFify I use the PDFView provided by macOS. It shoud be the same as used in Apple's "Preview" app. 

@TLMurray23 could you please try to reproduce this error in Apple's "Preview" app. If it is reproducible there too, then it is very likely a problem that originates from Apple, which sadly then is not in my hands to be solved. Thanks.

**Comment by TLMurray23** [comment]
Yeah, it's an Apple thing. The behavior in Preview is the same as PDFify. 

In Adobe Acrobat, the paste does not retain the formatting of what's on the clipboard. 

But I do wish PDFify did something that many Apple apps do: Shift+Option+Comand+V pastes the clipboard text only -- no formatting. 

Thanks for your reply.

**Comment by holtwick** [comment]
> But I do wish PDFify did something that many Apple apps do: Shift+Option+Comand+V pastes the clipboard text only -- no formatting.

I see, that's a great idea!

**Comment by TLMurray23** [comment]
Interestingly -- and annoyingly -- while many Apple apps do that paste-as-text thing, Preview does not! It doesn't paste anything at all with that keystroke.
